dir.traversal.targets

```diff
@@ -6,7 +6,7 @@
     <!-- To Serialize we use msbuild's batching functionality '%' to force it to batch all similar projects with the same identity 
          however since the project names are unique it will essentially force each to run in its own batch -->
     <MSBuild Targets="Build" Projects="@(Project)" Condition="'$(SerializeProjects)'=='true'" Properties="Dummy=%(Identity)"/>
-    <MSBuild Targets="Build" Projects="@(Project)" Condition="'$(SerializeProjects)'!='true'" BuildInParallel="true" />
+    <MSBuild Targets="Build" Projects="@(Project)" Condition="'$(SerializeProjects)'!='true'" BuildInParallel="false" />
```

Comment:
We have a bug in msbuild 2017. It is getting failed if we use parallel build. As soon as msbuild will be fixed we can turn it on.